[2024-08-11T10:38:25.801+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2024-08-11T10:38:21.137067+00:00 [queued]>
[2024-08-11T10:38:25.826+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2024-08-11T10:38:21.137067+00:00 [queued]>
[2024-08-11T10:38:25.827+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-08-11T10:38:25.828+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-08-11T10:38:25.828+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-08-11T10:38:25.858+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-08-11 10:38:21.137067+00:00
[2024-08-11T10:38:25.868+0000] {standard_task_runner.py:55} INFO - Started process 414 to run task
[2024-08-11T10:38:25.879+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2024-08-11T10:38:21.137067+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp3h4auq8b']
[2024-08-11T10:38:25.883+0000] {standard_task_runner.py:83} INFO - Job 18: Subtask is_api_available
[2024-08-11T10:38:26.032+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2024-08-11T10:38:21.137067+00:00 [running]> on host ac5f4abca0b6
[2024-08-11T10:38:26.195+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2024-08-11T10:38:21.137067+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-11T10:38:21.137067+00:00
[2024-08-11T10:38:26.199+0000] {http.py:120} INFO - Poking: api/
[2024-08-11T10:38:26.227+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2024-08-11T10:38:26.229+0000] {http.py:148} INFO - Sending 'GET' to url: http://randomuser.me/api/
[2024-08-11T10:38:27.890+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2024-08-11T10:38:27.913+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20240811T103821, start_date=20240811T103825, end_date=20240811T103827
[2024-08-11T10:38:27.984+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-08-11T10:38:28.068+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
